go/cmd/gh-ost/main.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/github/gh-ost/go/base"
 	"github.com/github/gh-ost/go/logic"
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
Added this to make it clear that gh-ost was registering the `go-sql-driver/mysql` driver as its `mysql` driver. We had some confusion previously and thought that the driver might have been coming from the `siddontang/go-mysql` library referenced in the `go/binlog/gomysql_reader.go`. But, the driver was in fact the `go-sql-driver/mysql`, pulled in though an import statement located within `github.com/outbrain/golib/sqlutils`. Hoping that adding the import directly here instead of relying on the indirect import will make it more clear for others.